### DISCUSSION TITLE
Problem with the to-do list

### DISCUSSION THREAD

**Discussion by KingerNL** [community/community#43562] [GitHub Education]
Hi guys,
I was wondering if I'm doing anything wrong to check off the to-do list item; "Explore a Student Pack offer", it looks like this:
![image](https://user-images.githubusercontent.com/67836542/211147291-98b4d16e-e499-4da0-a941-365ebf57a120.png)

I found online you should have the Student Pack if your logged in, like this: 
![image](https://user-images.githubusercontent.com/67836542/211147304-f4ef4a97-2e8a-4b45-8a54-70da8fed0a07.png)

But it doesn't seem to work, am I doing something wrong?

**Comment by melvincwng** [comment]
@KingerNL, I'm also facing the same issue as well.

Seems like navigating to other options/pages on the GitHub Education website doesn't check the last 2 "To-do" list items.

It's something I've observed for the past few months as well. Previously I was also trying to finish the "To-do" list items, but couldn't do so. Might be a potential bug 🤔?

Hope this information helps you out anyway 😄

**Reply by KingerNL** [reply]
@melvincwng thanks for letting me know I'm not the only one 🙂. 

It worries me that it's been a couple of months, was hoping I overlooked something 😅

**Reply by melvincwng** [reply]
@KingerNL, I have submitted a bug report to the GitHub Support team. 

Let's hope one of the support engineers can check & resolve this issue 🙏🏻 

FYI.
